tests/unit_tests/job_queue/test_realization_state.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+from pathlib import Path
+
+from ert.config import QueueConfig, QueueSystem
+from ert.job_queue import QueueableRealization, RealizationState
+from ert.run_arg import RunArg
+
+
+async def test_that_submit_propagates_to_driver(tmpdir, monkeypatch):
```

Comment:
When having `test_that_submit_propagates_to_driver` is `test_that_driver_submit_triggers_statechange_accept` required?

Reply:
I split this test in two to separate the functionality provided by the driver and functionality provided by the state machine.